google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -150,6 +149,7 @@
       Metadata.Key.of("x-goog-request-params", Metadata.ASCII_STRING_MARSHALLER);
   private static final Logger log = Logger.getLogger(ITStorageTest.class.getName());
   private static final String BUCKET = RemoteStorageHelper.generateBucketName();
```

Comment:
You might simply need more instance variables and fewer static ones. BUCKET should probably be an instance variable.

Reply:
BUCKET is a constant standing for a bucket name shared across multiple tests. There is no problem with sharing variables within the VM, some but cases might intersect using a shared resource in the Storage.

Reply:
yes, and that's why we should use a fresh bucket for each test. I think getting a new name each time will suffice.

Reply:
Yes, a fresh bucket per test will certainly help, but I'm not sure we can afford to create so many buckets. ITStorageTest often fails because of `exceeded rate limit to bucket creation`